KDagal-HTML-and-CSS-Week1/css/style.css

```diff
@@ -0,0 +1,34 @@
+/* Primary header*/
+header {
+    color: yellowgreen;
+    text-align: center;
+}
+
+/* Image*/
+img{
+    width: 97%;
+    padding: 10px;
+}
+
+/* header 2*/
+h2{
+color: whitesmoke;
+}
+
+/* Body of the page*/
+body{
+    background-color: rgb(47, 230, 47);
+}
+
+/* first paragraph is hilghited*/
+section :first-child{
+    background-color: burlywood;
+}
+
```

Comment:
Just noticed that the first child of each of the articles is also inheriting this background colour, if you want to avoid that you could add `article :first-child{ background-color: green}` afterwards